school-manager-server/pom.xml

```diff
@@ -87,6 +94,16 @@
                     <skipTests>true</skipTests>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.flywaydb</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <version>${flyway.version}</version>
+                <configuration>
+                    <user>root</user>
+                    <password>root</password>
+                    <url>jdbc:mysql://localhost:3306/managerschooldb</url>
+                </configuration>
+            </plugin>
```

Comment:
Why did you decide to migrate by plugin if spring boot has build in functionality for migration ? 
You plan to leave login/password in pom file in production ?